internal/sdkv2provider/schema_cloudflare_ruleset.go

```diff
@@ -93,6 +93,19 @@ func resourceCloudflareRulesetSchema() map[string]*schema.Schema {
 						Optional:    true,
 						Description: "Brief summary of the ruleset rule and its intended use.",
 					},
+					"last_updated": {
+						Type:        schema.TypeString,
+						Optional:    true,
+						Description: "The most recent update to this rule.",
+					},
+					// "categories": {
```

Comment:
is this intentional? unsure if we wanted to introduce this breaking change here.